database/dummy.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
+const momentRandom = require('moment-random');
+
+const restaurantName = ['A Salt & Battery', 'Party Fowl', 'Thai Me Up Restaurant & Brewery', 'Wild Thyme Café', 'Tequila Mockingbird', 'Life of Pie', 'Sconehenge', 'The Codfather', 'The Notorious P.I.G', 'Pig’N Pancake', 'Backyard Bowls', 'The Little Easy', 'Pies n’ Thighs', 'Pot Au Feu', 'Cibo Matto', 'Siena', 'The Dorrance', 'The French Laundry', 'Dining Divas', 'Kitchen Corral', 'Uzen', 'Crystal Fish', 'Okaine', 'Chicken Things', 'Acapulco Gold', 'Achilles', 'Acorn Palace', 'Affecté', 'Affluential', 'Bleeko', 'Big Juicy', 'Bella Figala', 'Cafe Constant', 'Capricorn', 'Club Soma', 'Classio', 'Crab and Prawn', 'Crustacia', 'Dutch Oven Gretel', 'Eat Underground', 'Escargot', 'Food Frolic', 'Funky Chicken Soup', 'Gelatosphere', 'Godzilla Pizza', 'Homeroom Cafe', 'Hot Sour and Soar', 'Kaku', 'La Zilla', 'Mongoliant Little Sheep', 'Left Blank', 'Lizzie Bordens', 'Mint Dynasty', 'Mongerfish', 'Netplosion', 'New Orlean', 'Nibbles', 'Nob Hill', 'Nosh on the Net', 'Nostrilé', 'Passion Fruit', 'Peace Meal', 'Pastor Pizza', 'Radiant Fresh', 'Kurasabe', 'Raja', 'Rococo', 'Royal Lush', 'Saguaro', 'SeaSky', 'Shitaki', 'Seafood Seranade', 'Scrap Opera', 'Té Tango', 'Tequila Mockingbird', 'Thalia', 'The 4AM Club', 'The Beef Plank', 'The Crawfish Cantina', 'The Crustacean', 'The Potato Constant', 'Torpor Ritz', 'Troché', 'Waffle Stomper', 'Wasabi Fresh', 'Wire House', 'Zen BBQ', 'Zoo Stew', 'Zuke', 'Song Bird', 'Zilladelphia', 'Dinner in the Sky', ' Sur un Arbre Perché', 'Normas', ' Devil Island Prison Restaurant ', 'Ninja Dining', 'Aurum', 'Homewood Mountain Resort', 'Mountain Retreat', 'The Five Fishermen'];
+
+const times = ['17:00', '17:15', '17:30', '17:45', '18:00', '18:15', '18:30', '18:45', '19:00', '19:15', '19:30', '19:45', '20:00', '20:15', '20:30', '20:45', '21:00', '21:15', '21:30', '21:45', '22:00', '22:15'];
```

Comment:
Are times for each restaurant all the same? Maybe it could be more "random" if some restaurants were open earlier in the day, etc..